portals/publisher/source/src/app/components/Apis/Details/APIDefinition/ServiceEndpoint.jsx

```diff
@@ -0,0 +1,200 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { useState } from 'react';
+import {
+    Grid,
+    Tooltip,
+    InputAdornment,
+    withStyles,
+    IconButton, Icon,
+} from '@material-ui/core';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import TextField from '@material-ui/core/TextField';
+import Autocomplete from '@material-ui/lab/Autocomplete';
+import { isRestricted } from 'AppData/AuthManager';
+
+
+const styles = (theme) => ({
+    endpointInputWrapper: {
+        width: '100%',
+        display: 'flex',
+        justifyContent: 'space-between',
+    },
+    textField: {
+        width: '100%',
+    },
+    input: {
+        marginLeft: theme.spacing(1),
+        flex: 1,
+    },
+    iconButton: {
+        padding: theme.spacing(1),
+    },
+
+});
+
+// eslint-disable-next-line valid-jsdoc
```

Comment:
Do not disable this rule, it's just a warning and you don't have to suppress rule

Reply:
Fixed in https://github.com/wso2/apim-apps/pull/144/commits/8e44f8a31d9ed97378691dd56ee3967f3ff554a2